profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/ETW/IpcServer.cpp

```diff
@@ -30,6 +31,11 @@ IpcServer::~IpcServer()
 
 void IpcServer::Stop()
 {
+    // Stop could be called when error and in the destructor
+    if (_stopRequested.load())
+    {
+        return;
+    }
     _stopRequested.store(true);
```

Comment:
Can `Stop` be called by at least 2 different threads ~samish time ?
if yes, maybe use something like 
``` cpp
auto stopRequested = _stopRequested.exchange(true);
if (stopRequested)
{
...
```

Reply:
✅ done